kv.go

```diff
@@ -69,6 +69,9 @@ type KeyValue interface {
 	Bucket() string
 	// PurgeDeletes will remove all current delete markers.
 	PurgeDeletes(opts ...WatchOpt) error
```

Comment:
What do we use the WatchOpt for mostly here? Just context?

Reply:
We pass whatever watch options that Watch() can accept, including context, but yeah. Don't know if that made sense or not, but that's how it is right now.